.github/workflows/deploy-dev.yml

```diff
@@ -122,6 +132,29 @@ jobs:
       WFNEWS_BASEURL: https://wfnews-api.dev.bcwildfireservices.com/
       WFNEWS_QUEUESIZE: 1500
 
+      #WFONE-PUSH-NOTIFICATION-API VARIABLES
+      WFONE_PUSH_NOTIFICATION_DATASOURCE_URL: ${{ inputs.wfone_pgsqlDB.endpoint }}
+      WFONE_PUSH_NOTIFICATION_DATASOURCE_USERNAME: "proxy_wf1_notification_rest_int"
+      WFONE_PM_SQS_S3_BUCKET_NAME: ${{ inputs.wfnews-monitor-queue-bucket.id }}
+      WFONE_PUSH_NOTIFICATION_MAX_CONNECTIONS: 25
+      WFONE_PUSH_NOTIFICATION_EXPIRE_HOURS: 24
+      WFONE_SQS_QUEUE_NOTIFICATION_URL: ${{ inputs.queue_notifications.id }}
+      WFONE_SQS_QUEUE_NOTIFICATION_URL_ACTIVE_FIRES: ${{ inputs.queue_fires.id }}
+      WFONE_SQS_QUEUE_NOTIFICATION_URL_BANS_PROHIBITIONS: ${{ inputs.queue_bans.id }}
+      WFONE_SQS_QUEUE_NOTIFICATION_URL_AREA_RESTRICTIONS: ${{ inputs.queue_restrictions.id }}
+      WFONE_SQS_QUEUE_NOTIFICATION_URL_EVAC_ORDERS: ${{ inputs.queue_evacs.id }}
+      WFONE_PUSH_NOTIFICATION_SQS_MONITOR_ATTRIBUTE: monitorType
+      WFONE_PUSH_NOTIFICATION_SQS_MAX_MESSAGES: 10
+      WFONE_PUSH_NOTIFICATION_SQS_WAIT_SECONDS: 20
+      WFONE_PUSH_NOTIFICATION_CONSUMER_INTERVAL_SECONDS: 120
+      WFONE_PUSH_ITEM_EXPIRE_HOURS_EVAC: 24
+      WFONE_PUSH_ITEM_EXPIRE_HOURS_FIRE: 48
+      WFONE_PUSH_ITEM_EXPIRE_HOURS_BAN: 8760
+      WFONE_PUSH_ITEM_EXPIRE_HOURS_RESTRICTED_AREA: 8760
+      WFONE_FIREBASE_DB_URL: https://wildlife-push-test.firebaseio.com
```

Comment:
We should probably move a bunch of these configs into vars, just so we can change them without PRing changes

Reply:
and the [INT] one should be removed and replaced with the environment var